open_humans/templates/partials/import-button.html

```diff
@@ -2,7 +2,7 @@
   {% if user.member.primary_email.verified %}
     {% if source %}
     <a class="btn btn-primary {{ class }}"
-      href="{{ source.href_connect }}?origin=open-humans&next={{ source.href_next }}">
+      href="{{ source.href_connect }}?origin=open-humans&amp;next={{ source.href_next }}">
```

Comment:
I changed it because the HTML linting my Atom was using flagged it. This seems to indicate it's correct? http://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href